scripts/uberenv/spack_configs/toss_3_x86_64_ib/compilers.yaml

```diff
@@ -6,11 +6,24 @@ compilers:
     modules: []
     operating_system: rhel7
     paths:
-      cc:  /usr/tce/packages/gcc/gcc-4.9.3/bin/gcc
-      cxx: /usr/tce/packages/gcc/gcc-4.9.3/bin/g++
+      cc:  /usr/tce/packages/clang/clang-4.0.0/bin/clang
+      cxx: /usr/tce/packages/clang/clang-4.0.0/bin/clang++
       f77: /usr/tce/packages/gcc/gcc-4.9.3/bin/gfortran
```

Comment:
Should the gfortran version be updated as well?

(If the answer is yes, it could also be done separately)

Reply:
<shrugs>